app/assets/javascripts/Results/context_menu.js

```diff
@@ -11,6 +11,24 @@
 var annotation_context_menu = {
   setup: function () {
     var menu_items = {
+      check_mark_annotation: {
+        title: "✅",
+        cmd: "check_mark_annotation",
+        action: () => resultComponent.addQuickAnnotation("✅"),
+        addClass: "check_mark_annotation",
```

Comment:
You can just use the same class name (e.g. `emoji-annotation-context-menu-item`) for all of the new items, since the styling is the same.

(For new classes hyphens are preferred to underscores.)